Userland/Libraries/LibGfx/ImageFormats/JPEGXLLoader.cpp

```diff
@@ -1365,7 +1365,7 @@ class SelfCorrectingData {
 ///
 
 /// H.2 - Image decoding
-struct ModularHeader {
+struct ModularData {
```

Comment:
Commit description: "so let's rename also rename it." has too many verbs.

(Sorry about missing this this morning.)